View Controllers/ComposeViewController.m

```diff
@@ -0,0 +1,57 @@
+//
+//  ComposeViewController.m
+//  twitter
+//
+//  Created by Sarah Wang on 6/29/21.
+//  Copyright © 2021 Emerson Malca. All rights reserved.
+//
+
+#import "ComposeViewController.h"
+#import "APIManager.h"
+
+@interface ComposeViewController ()
+@property (weak, nonatomic) IBOutlet UITextView *postView;
+
+@end
+
+@implementation ComposeViewController
+
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+}
+- (IBAction)didTweet:(UIBarButtonItem *)sender {
```

Comment:
This can use 'id'.

Questions. What is `id` in ObjC? Why is it okay to use that here instead of `UIBarButtonItem`?